flathunter/config.py

```diff
@@ -16,7 +17,8 @@ class Config:
     __searchers__ = [CrawlImmobilienscout(),
                      CrawlWgGesucht(),
                      CrawlEbayKleinanzeigen(),
-                     CrawlImmowelt()]
+                     CrawlImmowelt(),
+                     CrawlIdealista()]
```

Comment:
Also, if you're adding a new crawler, it would make sense to update the README - there's a couple of places where we list these.

Reply:
README updated with some additions and comments on the new crawler